testscommon/hyperBlocksPoolStub.go

```diff
@@ -0,0 +1,85 @@
+package testscommon
+
+import (
+	"github.com/multiversx/mx-chain-ws-connector-firehose-go/data"
+	"github.com/multiversx/mx-chain-ws-connector-firehose-go/data/hyperOutportBlocks"
+)
+
+// HyperBlocksPoolStub -
+type HyperBlocksPoolStub struct {
```

Comment:
I think this should be renamed to HyperBlocksPoolMock as per latest discussions.. same for all Stubs

Reply:
renamed